### PR TITLE
resolve lint warnings

### PR DESCRIPTION
**Pull request by hodamohammadi**
Resolving lint warnings and using class names as TAG

### CODE REVIEW

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt [hunk 1]
```diff
@@ -8,10 +8,12 @@ import java.util.Date
 /**
  * Default model for a single message.
  */
-data class DefaultMessage(val messageId: String,
-                          val messageText: String,
-                          val messageCreatedAt: Date,
-                          val messageUser: DefaultUser) : Serializable, IMessage {
+data class DefaultMessage(
+        val messageId: String,
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt#L12"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt [hunk 2]
```diff
@@ -8,10 +8,12 @@ import java.util.Date
...
+        val messageText: String,
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt#L13"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt [hunk 3]
```diff
@@ -8,10 +8,12 @@ import java.util.Date
...
+        val messageCreatedAt: Date,
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt#L14"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt [hunk 4]
```diff
@@ -8,10 +8,12 @@ import java.util.Date
...
+        val messageUser: DefaultUser
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt#L15"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt [hunk 5]
```diff
@@ -8,10 +8,12 @@ import java.util.Date
...
+): Serializable, IMessage {
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt:16]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/cc852f47c5e6c83c4f338fe315a527af912ea1ff/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultMessage.kt#L16"></span>Missing spacing before “:”</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt [hunk 6]
```diff
@@ -6,9 +6,11 @@ import java.io.Serializable
 /**
  * Default model for a user.
  */
-data class DefaultUser(val userId: String,
-                       val userName: String?,
-                       val userAvatar: String?) : Serializable, IUser {
+data class DefaultUser(
+        val userId: String,
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt#L10"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt [hunk 7]
```diff
@@ -6,9 +6,11 @@ import java.io.Serializable
...
+        val userName: String?,
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt#L11"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt [hunk 8]
```diff
@@ -6,9 +6,11 @@ import java.io.Serializable
...
+        val userAvatar: String?
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt#L12"></span>Unexpected indentation (expected 4, actual 8)</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt [hunk 9]
```diff
@@ -6,9 +6,11 @@ import java.io.Serializable
...
+): Serializable, IUser {
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt:13]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/cc852f47c5e6c83c4f338fe315a527af912ea1ff/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt#L13"></span>Missing spacing before “:”</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/services/ConveyorNotificationService.kt [hunk 10]
```diff
@@ -3,5 +3,5 @@ package com.hodamohammadi.conveyor.services
 import com.google.firebase.messaging.FirebaseMessagingService
 
 class ConveyorNotificationService: FirebaseMessagingService() {
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/services/ConveyorNotificationService.kt:5]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/cc852f47c5e6c83c4f338fe315a527af912ea1ff/conveyor/src/main/java/com/hodamohammadi/conveyor/services/ConveyorNotificationService.kt#L5"></span>Missing spacing before “:”</td>
    </tr>
  </tbody>
</table>

File: conveyor/src/main/java/com/hodamohammadi/conveyor/services/ConveyorNotificationService.kt [hunk 11]
```diff
@@ -3,5 +3,5 @@ package com.hodamohammadi.conveyor.services
...
-
+    //TODO: implement methods.
```
Comment (ReviewerOfCode) [inline conveyor/src/main/java/com/hodamohammadi/conveyor/services/ConveyorNotificationService.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hodamohammadi/Conveyor/blob/55bee9f1f017d0be288727d3f9aed6702492de72/conveyor/src/main/java/com/hodamohammadi/conveyor/services/ConveyorNotificationService.kt#L6"></span>Missing space after //</td>
    </tr>
  </tbody>
</table>

**Comment by ReviewerOfCode** [comment]
### AndroidLint found issues

#### Warning (2)

| File | Line | Reason |
| ---- | ---- | ------ |
`conveyor/build.gradle` | 12 | Not targeting the latest versions of Android; compatibility modes apply. Consider testing and updating this version. Consult the android.os.Build.VERSION_CODES javadoc for details. 
`conveyor/src/main/AndroidManifest.xml` | 6 | On SDK version 23 and up, your app data will be automatically backed up, and restored on app install. Your GCM regid will not work across restores, so you must ensure that it is excluded from the back-up set. Use the attribute `android:fullBackupContent` to specify an `@xml` resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by itsAlexNguyen — APPROVED** [review]

**Pull request merged**